.gitignore

```diff
@@ -16,6 +16,8 @@
 
 *.class
 *.log
+.DS_Store
+src/main/resources/typescriptc.js
```

Comment:
You are right, after I put the compilation result into the target - we don't need this anymore